ci-operator/config/openshift-kni/cnf-features-deploy/openshift-kni-cnf-features-deploy-master.yaml

```diff
@@ -44,6 +40,12 @@ images:
   from: base
   to: dpdk
 promotion:
+  excluded_images:
```

Comment:
when I see correct, all images are excluded now? I know this has been an issue, not sure if it was fixed. If something does not work as expected, you might need to remove the promotion section.

Reply:
If I remove the promotion sections all the images are promoted anyway using the tag_specification stream (see https://github.com/openshift/ci-tools/blob/master/CONFIGURATION.md#promotion) .

The issue here is that both master and release-4.5 are pointing to the 4.5 stream, so images would be promoted from two different branches (and also, CI was smart enough to raise that as an error, so I had to add this).

Reply:
oh, good to know there is some default promotion, missed it :+1: